public/app/features/dashboard-scene/scene/DashboardControlsUrlSync.tsx

```diff
@@ -0,0 +1,39 @@
+import { SceneObjectUrlSyncHandler, SceneObjectUrlValues } from '@grafana/scenes';
+
+import { DashboardControls } from './DashboardControls';
+
+export class DashboardControlsUrlSync implements SceneObjectUrlSyncHandler {
```

Comment:
any reason for using a special class here and not just SceneObjectUrlSyncConfig. ?

Reply:
Removed! I thought it was cleaner, it is not needed :)